aa-sdk/core/src/middleware/defaults/gasEstimator.ts

```diff
@@ -60,6 +60,8 @@ export const defaultGasEstimator: <C extends MiddlewareClient>(
       );
       (struct as UserOperationStruct<"0.7.0">).paymasterVerificationGasLimit =
         paymasterVerificationGasLimit;
+
+      (struct as UserOperationStruct<"0.7.0">).paymasterPostOpGasLimit = "0x0";
```

Comment:
When is this supposed to return a non dummy value? From sponsorship service?

Reply:
yes, pm_getPaymsterStubData will return an optional `paymasterPostOpGasLimit` value. we don't do that in our impl.